.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
73 changed files only for email service? Please close this merge request and create new one. Correct one.